analyzers/src/ITs.JsonParser/Json/RuleIssues.cs

```diff
@@ -30,16 +30,19 @@ public class RuleIssue
 {
     public string Id { get; set; }
     public string Message { get; set; }
-    public SarifLocationFile Location { get; set; } // TODO: This should not re-use SarifLocationFile type.
+    public string Uri { get; set; }
+    public string Location { get; set; }
 
     public RuleIssue(SarifIssue issue)
     {
         Id = issue.RuleId;
         Message = issue.Message;
-        Location = new()
+        Uri = issue.NormalizedUri();
+        if (issue.Location is { } location)
         {
-            Uri = issue.NormalizedUri(),
-            Region = issue.Location?.Region
-        };
+            Location = location.Region.StartLine == location.Region.EndLine
+                ? $"Line(s) {location.Region.StartLine} Position {location.Region.StartColumn}#{location.Region.EndColumn}"
+                : $"Line(s) {location.Region.StartLine}-{location.Region.EndLine} Position {location.Region.StartColumn}#{location.Region.EndColumn}";
```

Comment:
Final state should be ("s", "region", "-")
```suggestion
                : $"Lines {region.StartLine}-{region.EndLine} Position {region.StartColumn}-{region.EndColumn}";
```

Reply:
I do understand the proposal in the code but I don't fully understand the comment `Final state should be ("s", "region", "-")`.

Reply:
It explicitly mentiones all the changes in proposal, so none of them is missed. They are all easy to overlook